openraft/src/core/raft_core.rs

```diff
@@ -1368,25 +1471,13 @@ impl<C: RaftTypeConfig, N: RaftNetworkFactory<C>, S: RaftStorage<C>> RaftRuntime
                 debug_assert!(got.is_some(), "there has to be a buffered snapshot data");
             }
             Command::InstallSnapshot { snapshot_meta } => {
-                let snapshot_data = self.received_snapshot.remove(&snapshot_meta.snapshot_id);
+                let data = self.received_snapshot.remove(&snapshot_meta.snapshot_id).unwrap();
```

Comment:
should add some comments on `unwrap()` call or use `?` instead.

Reply:
Some comments would be good.

If there is no such `snapshot_id`, it is a bug thus I just panic it.